### PR TITLE
chore(ng): backport #2919 to v18.1

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## Description

As #2919 is not breaking and provide great i18n improvement, we add it on v18.1 branch too.

-----

Optionally, technical or more in-depth description for reviewers.
Keep an empty line under your text, as well as the 5 lines that follow it.

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2968/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/20804/ui-diff)

File: packages/ng/date/select/translations.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+﻿export const Translations = {
```
Comment (shajz) [inline packages/ng/date/select/translations.ts:1]: These are the same as the date input translations. Maybe we could mutualize them later :)

**Review by shajz — APPROVED** [review]

File: packages/ng/date/select/translations.ts (hunk 1 above)
Comment (GuillaumeNury) [reply packages/ng/date/select/translations.ts:1]: A lot of translations are duplicated we could improve that!

Comment (shajz) [reply packages/ng/date/select/translations.ts:1]: Yeah, I noticed that further in the review. Let's do that later©

**Review by shajz — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2968/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/20825/ui-diff)

**Pull request merged**